src/api/user.test.tsx

```diff
@@ -68,3 +68,24 @@ describe('useEditUser', () => {
     'sends axios request to edit a user and throws an appropriate error on failure'
   );
 });
+
+describe('useDeleteUser', () => {
+  it('delete request to delete user and returns successful response', async () => {
+    const { result } = renderHook(() => useDeleteUser(), {
+      wrapper: hooksWrapperWithProviders(),
+    });
+    expect(result.current.isIdle).toBe(true);
+
+    result.current.mutate('user1');
+
+    await waitFor(() => {
+      expect(result.current.isSuccess).toBeTruthy();
+    });
+
+    expect(result.current.data).toEqual('');
+  });
+
+  it.todo(
+    'sends axios request to delete user session and throws an appropriate error on failure'
+  );
```

Comment:
Either implement the test or delete

Reply:
The errors are handle in the mutateAsync. This shouldn't have been added in. I'll remove it